R/serializer.R

```diff
@@ -103,7 +105,7 @@ as_attachment <- function(value, filename = NULL) {
 #'
 #' Serializers are used in Plumber to transform the R object produced by a
 #' filter/endpoint into an HTTP response that can be returned to the client. See
-#' [here](https://book.rplumber.io/articles/rendering-output.html#serializers-1) for
+#' [here](../articles/rendering-output.html#serializers-1) for
```

Comment:
CRAN will believe the relative link is an error. 😒 

```suggestion
#' [here](https://www.rplumber.io/articles/rendering-output.html#serializers-1) for
```

Reply:
Oh well